Add docker-compose for development environment

kei178 opened the pull request:
Add docker-compose for the development setup.

kei178 commented:
@naveenbalachandar - Could you also take a look at this, please?

kei178 commented:
I added phpMyAdmin to the docker-compose. The login credentials are described in the top comment.

naveenbalachandar commented on docker-compose.yml:
```diff
@@ -0,0 +1,48 @@
+      - 'moodle_data:/bitnami/moodle'
+      - 'moodledata_data:/bitnami/moodledata'
+      # Install Accredible plugin by default
+      - '.:/bitnami/moodle/mod/accredible'
```
As discussed, there is an issue with adding the accredible plugin on a new container. The moodle service fails to start in this scenario.

kei178 commented:
## What additionally updated

- Added the "Development Information" section to `README.md`
- Added vim to the moodle service.
- Separated the original `docker-compose.yml` into `docker-compose.yml` and `docker-compose.plugin.yml` which includes additional config for the plugin.

kei178 commented:
Added instructions for a common setup error to README.md.
